src/app/output/preset-output-files/preset-output-files.component.ts

```diff
@@ -58,42 +57,31 @@ export class PresetOutputFilesComponent implements OnInit {
     );
   }
 
-  private sendPresetStatus(): void {
-    let totalTasks = this.tasks.length;
-    let finishedTasks = 0;
-
-    for(const taskInfo of this.tasks) {
-        if(taskInfo.status === Status.Finished.valueOf()) {
-            finishedTasks += 1;
-        }
-    }
-    this.numberFinishedTasksEvent.emit(new TranscodingPresetStatus(this.format, this.ffmpegVideoCodec, finishedTasks, totalTasks));
-  }
-
   private updateTasksStatuses(): void {
     const taskIds = this.storageService.getTaskIds();
-    let finished = true;
-    if(taskIds && taskIds[this.format] && taskIds[this.format][this.ffmpegVideoCodec]) {
+    let finishedTasks = 0;
+    let tasksNumber = 0;
+    if (taskIds && taskIds[this.format] && taskIds[this.format][this.ffmpegVideoCodec]) {
       const tasks = taskIds[this.format][this.ffmpegVideoCodec];
-      finished = finished && (tasks.length === this.tasks.length);
+      tasksNumber = tasks.length;
       for (const taskEntity of tasks) {
         const taskInfo: TaskInfo = this.getTaskInfoEntity(taskEntity.res);
         this.updateTaskStatus(taskEntity.id, taskInfo);
-        finished = finished && (taskInfo.status === Status.Finished.valueOf() || taskInfo.status === Status.Error.valueOf());
+        if (taskInfo.status === Status.Finished.valueOf() || taskInfo.status === Status.Error.valueOf()) {
```

Comment:
```suggestion
        if ([Status.Finished.valueOf(), Status.Error.valueOf()].includes(taskInfo.status)) {
```

or even keep a const somewhere else:
```
const FINISHED_STATUSSES = [Status.Finished.valueOf(), Status.Error.valueOf()];
```